pyqtorch/matrices.py

```diff
@@ -43,6 +42,10 @@
 )
 
 
+def PROJMAT(ket: Tensor, bra: Tensor) -> Tensor:
+    return torch.kron(ket, bra.T)
```

Comment:
pyq states have shape [2] * n_qubits + [batch_size] so this cant work

Reply:
Indeed, that's what I realised. Essentially, PyQ deals with single qubit operations.

Reply:
No, it just uses weird shapes 😅. I think all that is needed is to reshape the `ket` and `bra` here to essentially undo the `reshape` at the end of `product_state`, and then this kron will return the correct matrix.

Reply:
OK let me try it, thanks for the pointer.

Reply:
Done.